index.html

```diff
@@ -919,6 +919,93 @@ <h4>
   reader.scan({ recordType: "example.com:sp"});
       </pre>
     </aside>
+    <aside title="Push a smart poster message" class="example">
+      <p>
+        Smart poster defined in NFC Forum Smart Poster Record Type Definition
+        specification. It is an NDEF record that contains an <a>NDEF message</a>
+        as payload, which in turn may contain several records: a URI record that
```

Comment:
which at least contains the following records (need to check spec, maybe only URL is required)

Reply:
Yes, only URL is mandatory. We might want to standardize Sp in Web NFC indeed.

Reply:
More add an informal section describing it (as we were standardizing it, more or less)

Reply:
Let's handle the Sp in a next PR. I made the suggested changes, please re-check.